docs/environments.md

```diff
@@ -255,3 +255,9 @@ Learn more here: https://github.com/Wandercraft/jiminy
 An environment for behavioural planning in autonomous driving, with an emphasis on high-level perception and decision rather than low-level sensing and control. The difficulty of the task lies in understanding the social interactions with other drivers, whose behaviours are uncertain. Several scenes are proposed, such as highway, merge, intersection and roundabout.
 
 Learn more here: https://github.com/eleurent/highway-env
+
+### gym-carla: Gym Wrapper for CARLA Driving Simulator
+
+gym-carla provides a gym wrapper for the [CARLA simulator](http://carla.org/), which is a realistic 3D simulator for autonomous driving research. The environment includes a virtual city with several surrounding vehicles running around. Multiple source of observations are provided for the ego vehicle, such as front-view camera image, lidar point cloud image, and birdeye view semantic mask. Several applications have been developed based on this wrapper, such as deep reinforcement learning for end-to-end autonomous driving.
```

Comment:
ego vehicle - is that a typo? Otherwise, lgtm

Reply:
Thanks for review. "Ego vehicle" means the autonomous vehicle we are controlling, it's a commonly used term in autonomous driving community.

Reply:
👍